platforms/software/dependency-management/src/main/java/org/gradle/internal/component/resolution/failure/ResolutionFailureHandler.java

```diff
@@ -243,6 +243,7 @@ private AbstractResolutionFailureException reportExceptionAsProblem(AbstractReso
             ResolutionFailureProblemId problemId = exception.getFailure().getProblemId();
             builder.id(TextUtil.screamingSnakeToKebabCase(problemId.name()), problemId.getDisplayName(), GradleCoreProblemGroup.variantResolution())
                 .contextualLabel(exception.getMessage())
+                .details(exception.getFailure().describeRequestTarget())
```

Comment:
@gradle/bt-jvm fyi

Reply:
❗ I don't think you want to use this.  The target can vary in both type and format based on the failure type.

The javadoc on `details` says "The long description of this problem."  I think. `exception.getFailure().getProblemId().getDisplayName()` might be more what you're looking for.